tests/test_integration_ocr.py

```
@@ -0,0 +1,51 @@
+    manifest_file = tmp_ep / 'manifest.jsonl'
+    with manifest_file.open('a') as fh:
+        fh.write(json.dumps(manifest_entry) + '\n')
+
+    # link environment variables so scripts pick up this epstein_project
+    env = os.environ.copy()
+    env['EPSTEIN_ROOT_OVERRIDE'] = str(tmp_ep)
+
+    # run the ocr runner with batch=1 but override paths by editing the script to point to tmp_ep
+    runner = Path(__file__).resolve().parents[1] / 'scripts' / 'ocr_runner.py'
+    # run with python
+    proc = subprocess.run(['python3', str(runner), '--batch', '1', '--min-bytes', '1', '--max-text-bytes', '100000'], env=env, stdout=subprocess.PIPE, stderr=subprocess.PIPE, timeout=120)
```

Ruff (0.14.10)

Prefer `capture_output` over sending `stdout` and `stderr` to `PIPE`

Replace with `capture_output` keyword argument

(UP022)